jdi-light/src/main/java/com/epam/jdi/light/driver/get/DriverData.java

```diff
@@ -262,6 +268,18 @@ public static void defaultEdgeOptions(EdgeOptions cap) { }
     public static void defaultOperaOptions(OperaOptions cap) { }
     public static JAction1<OperaOptions> OPERA_OPTIONS = DriverData::defaultOperaOptions;
 
+    public static void defaultAndroidOptions(MutableCapabilities cap) {
+        // Capabilities from settings
+        CAPABILITIES_FOR_ANDROID.forEach(cap::setCapability);
+    }
+    public static JAction1<MutableCapabilities> ANDROID_OPTIONS = DriverData::defaultAndroidOptions;
+
+    public static void defaultIOSOptions(MutableCapabilities cap) {
+        // Capabilities from settings
+        CAPABILITIES_FOR_IOS.forEach(cap::setCapability);
+    }
+    public static JAction1<MutableCapabilities> IOS_OPTIONS = DriverData::defaultIOSOptions;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5104364)